### ISSUE TITLE
DRAFT: Explore grid layer render error

### ISSUE THREAD

**Issue by jwalgran** [open-apparel-registry/open-apparel-registry#816]
# Draft

This is a draft issue, and it will need to clarify specs and formalize what
counts as "done" before work begins on it.

Check with the issue's creator before starting work on this issue.

## Overview

A clear description of the feature request.

### Describe the solution you'd like

- Experiment with using web mercator in ST_AsMVTGeom call
- Experiment with ST_AsMVTGeom buffer size (try reducing and expanding)

Time box this to no more than a day.

### Additional Context
<img width="1040" alt="Screen Shot 2019-09-18 at 12 45 45 PM" src="https://user-images.githubusercontent.com/17363/65180677-4ac1eb80-da12-11e9-85f7-70188b0afac2.png">

**Comment by jwalgran** [comment]
Duplicate of #801
